### PR TITLE
Add several missing translations and improve few previous translation for Spanish Locale

### PR DESCRIPTION
**Pull request by Lastimoso**
There where several missing translations in the "es" locale that causes a bad user experience and few "missing locale" errors.
This PR complete all missing translations and improve few others.

### CODE REVIEW

**Comment by sonalkr132** [comment]
Please rebase to fix the build.

**Comment by Lastimoso** [comment]
> Please rebase to fix the build.

Thanks @sonalkr132 I think I'va managed to do it. Please let me know if it works...

**Comment by sonalkr132** [comment]
Please don't merge master to your branch, instead rebase it https://git-scm.com/book/en/v2/Git-Branching-Rebasing
You could also just start over from master.

**Comment by Lastimoso** [comment]
> Please don't merge master to your branch, instead rebase it https://git-scm.com/book/en/v2/Git-Branching-Rebasing
> You could also just start over from master.
> 
> EDIT: use `git push origin update-spanish-locale -f`

I hope I got it right this time! Thanks for your patience!

**Comment by sonalkr132** [comment]
Looks great :+1: 
We just need someone who knows spanish to review this.

**Comment by Lastimoso** [comment]
Hi everyone. Is there anything I can do to help you move this forward? Let me know in the future if you need help with Spanish translation as I'm native speaker. Not to review my own work, of course :)

File: config/locales/es.yml [hunk 1]
```diff
@@ -1,74 +1,79 @@
 es:
   edit: Editar
-  failure_when_forbidden:
-  feed_latest: RubyGems.org | Gemas Nuevas
-  feed_subscribed: RubyGems.org | Subscripciones de Gemas
-  footer_about_html: RubyGems.org es el servicio de alojamiento de gemas de la comunidad de Ruby. Publica tus gemas de manera instantánea e instalalas. Usa la API para interactuar y buscar información de las gemas disponibles. Conviértete en contribuidor y mejora este sitio con tus cambios.
-  footer_sponsors_html:
-  footer_join_rt_html:
+  failure_when_forbidden: Por favor verifica la URL o intenta nuevamente.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  failure_when_forbidden: Por favor verifica la URL o inténtalo nuevamente.
```

File: config/locales/es.yml [hunk 2]
```diff
@@ -1,74 +1,79 @@
...
+  feed_latest: RubyGems.org | Gemas más recientes
+  feed_subscribed: RubyGems.org | Gemas suscritas
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  feed_subscribed: RubyGems.org | Suscripciones a gemas
```

File: config/locales/es.yml [hunk 3]
```diff
@@ -1,74 +1,79 @@
...
+  footer_about_html: RubyGems.org es el servicio de alojamiento de Gemas de la comunidad de Ruby. <a href="%{publish_docs}">Publica tus gemas</a> instantaneamente y luego <a href="%{install_docs}">instálalas</a>. Usa <a href="%{api_docs}">la API</a> para saber más sobre <a href="%{gem_list}">gemas disponibles</a>. <a href="%{contributing_docs}">Conviértete en colaborador</a> y mejora este sitio con tus cambios.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  footer_about_html: RubyGems.org es el servicio de alojamiento de Gemas de la comunidad de Ruby. <a href="%{publish_docs}">Publica tus gemas</a> instantáneamente y luego <a href="%{install_docs}">instálalas</a>. Usa <a href="%{api_docs}">la API</a> para saber más sobre <a href="%{gem_list}">las gemas disponibles</a>. <a href="%{contributing_docs}">Conviértete en colaborador</a> y mejora este sitio con tus cambios.
```

File: config/locales/es.yml [hunk 4]
```diff
@@ -1,74 +1,79 @@
...
+  footer_sponsors_html: RubyGems.org es posible gracias la colaboración de la fantástica comunidad de Ruby.
+    <a href="https://www.fastly.com/">Fastly</a> proveé ancho de banda y soporte de CDN
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
    <a href="https://www.fastly.com/">Fastly</a> provee ancho de banda y soporte de CDN
```

File: config/locales/es.yml [hunk 5]
```diff
@@ -1,74 +1,79 @@
...
+    <a href="http://www.rubycentral.org/">Ruby Central</a> cubre los costos de infraestructura y
+    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> financia el continuo desarrollo y trabajo en servidores.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> financia el desarrollo y el trabajo en los servidores.
```

File: config/locales/es.yml [hunk 6]
```diff
@@ -1,74 +1,79 @@
...
+    <a href="%{sponsors_page}">Aprende más sobre nuestros sponsors y como trabajan en conjunto</a>
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
    <a href="%{sponsors_page}">Aprende más sobre nuestros sponsors y cómo trabajan en conjunto</a>
```

File: config/locales/es.yml [hunk 7]
```diff
@@ -1,74 +1,79 @@
...
+  footer_join_rt_html: Necesitamos tu ayuda para financiar el tiempo de desarrolladores que mantiene RubyGems.org funcionando bien para todos.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  footer_join_rt_html: Necesitamos tu ayuda para financiar el tiempo de los desarrolladores que mantienen RubyGems.org funcionando correctamente para todo el mundo.
```

File: config/locales/es.yml [hunk 8]
```diff
@@ -1,74 +1,79 @@
...
+    <a href="https://rubytogether.org/join?source=rubygems">Unite a Ruby Together hoy</a>.
   form_disable_with: Un momento por favor...
   help_url: http://help.rubygems.org
-  invalid_page:
+  invalid_page: El número de página inexistente. Redireccionando a página por defecto.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  invalid_page: Número de página inexistente. Redireccionando a la página por defecto.
```

File: config/locales/es.yml [hunk 9]
```diff
@@ -1,74 +1,79 @@
...
+  footer_about_html: RubyGems.org es el servicio de alojamiento de Gemas de la comunidad de Ruby. <a href="%{publish_docs}">Publica tus gemas</a> instantaneamente y luego <a href="%{install_docs}">instálalas</a>. Usa <a href="%{api_docs}">la API</a> para saber más sobre <a href="%{gem_list}">gemas disponibles</a>. <a href="%{contributing_docs}">Conviértete en colaborador</a> y mejora este sitio con tus cambios.
+  footer_sponsors_html: RubyGems.org es posible gracias la colaboración de la fantástica comunidad de Ruby.
+    <a href="https://www.fastly.com/">Fastly</a> proveé ancho de banda y soporte de CDN
+    <a href="http://www.rubycentral.org/">Ruby Central</a> cubre los costos de infraestructura y
+    <a href="https://rubytogether.org/?source=rubygems">Ruby Together</a> financia el continuo desarrollo y trabajo en servidores.
+    <a href="%{sponsors_page}">Aprende más sobre nuestros sponsors y como trabajan en conjunto</a>
+  footer_join_rt_html: Necesitamos tu ayuda para financiar el tiempo de desarrolladores que mantiene RubyGems.org funcionando bien para todos.
+    <a href="https://rubytogether.org/join?source=rubygems">Unite a Ruby Together hoy</a>.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
    <a href="https://rubytogether.org/join?source=rubygems">Únete a Ruby Together hoy</a>.
```

File: config/locales/es.yml [hunk 10]
```diff
@@ -1,74 +1,79 @@
...
+  multifactor_authentication: Autenticación de múltiples factores
+  subtitle: el alojamiento de gemas de tu comunidad
+  this_rubygem_could_not_be_found: Esta gema no pudo encontrarse.
+  time_ago: "hace %{duration}"
   title: RubyGems.org
-  update: Actualizar
-  try_again:
-  advanced_search:
+  update: actualiza
+  try_again: Algo salió mal. Por favor intenta nuevamente.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
  try_again: Algo salió mal. Por favor inténtalo nuevamente.
```

File: config/locales/es.yml [hunk 11]
```diff
@@ -1,74 +1,79 @@
...
         handle: Usuario
         password: Contraseña
   dashboards:
     show:
       creating_link_text: crear
       gem_link_text: página de alguna gema
-      latest: Últimas Actualizaciones
-      latest_title: Feed RSS con últimas actualizaciones
+      latest: Últimas actulizaciones
+      latest_title: Feed RSS con últimas actulizaciones
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      latest_title: Feed RSS con últimas actualizaciones
```

File: config/locales/es.yml [hunk 12]
```diff
@@ -1,74 +1,79 @@
...
+        email: Dirección de correo
         handle: Usuario
         password: Contraseña
   dashboards:
     show:
       creating_link_text: crear
       gem_link_text: página de alguna gema
-      latest: Últimas Actualizaciones
-      latest_title: Feed RSS con últimas actualizaciones
+      latest: Últimas actulizaciones
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      latest: Últimas actualizaciones
```

File: config/locales/es.yml [hunk 13]
```diff
@@ -1,74 +1,79 @@
...
+      latest_title: Feed RSS con últimas actulizaciones
       migrating_link_text: migrar
       mine: Mis Gemas
-      my_subscriptions: Subscripciones
-      no_owned_html: Aún no has registrado ninguna gema. Tal vez puedes revisar las guías para %{creating_link} una gema o %{migrating_link} una gema de RubyForge.
-      no_subscriptions_html: Aún no te has subscrito a ningua gema. ¡Visita la %{gem_link} para subscribirte!
+      my_subscriptions: Suscripciones
+      no_owned_html: Todavía no publicaste ninguna gema. Quizás puedas ver las guiás sobre como %{creating_link} o %{migrating_link} una gema en RubyForge.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      no_owned_html: Todavía no publicaste ninguna gema. Prueba a ver las guiás sobre como %{creating_link} o %{migrating_link} una gema en rubygems.org.
```

File: config/locales/es.yml [hunk 14]
```diff
@@ -77,254 +82,254 @@ es:
...
     deletion_complete:
-      subject:
-      body_html:
+      subject: Tu cuenta se ha eliminado de rubygems.org
+      body_html: Se ha procesado tu solicitud para eliminar tu cuenta de rubygems.org. Siempre puedes crear una nueva cuenta en nuestra página %{sign_up}.
     deletion_failed:
-      subject:
-      body_html:
+      subject: Tu solicitud para eliminar tu cuenta de rubygems.org ha fallado
+      body_html: Has solicitado eliminar tu cuenta de rubygems.org. Lamentablemente, no hemos podido procesar tu pedido. Por favor intenta más adelante o %{contact} si el problema persiste.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      body_html: Has solicitado eliminar tu cuenta de rubygems.org. Lamentablemente, no hemos podido procesar tu pedido. Por favor inténtalo de nuevo más adelante o %{contact} si el problema persiste.
```

File: config/locales/es.yml [hunk 15]
```diff
@@ -77,254 +82,254 @@ es:
...
-      technical_html: 'Algunos aspectos técnicos del sitio: Es 100% Ruby. El sitio principal es una aplicación de %{rails}. Las gemas se almacenan en %{s3}, served by %{fastly}, y el tiempo entre que se publica una gema y que está lista para instalarse es mínimo. Para más información, %{source_code}, cuya licencia es %{license} en GitHub.'
+      support_html: A pesar de que RubiGems.org no es mantenido por una compañía en específico, muchas nos han ayudado. El diseño actual, las ilustraciones y el desarrollo de front-end del sitio fue cortesía de %{dockyard}. %{github} también ha sido invaluable por ayudarnos a colaborar y compartir código de manera sencilla. El sitio comenzó en %{heroku}, cuyo excelente servicio ayudó a probar que RubyGems es una solución viable en la que toda la comunidad puede confiar. Nuestra infraestructura está alojada actualmente en %{aws}.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      support_html: A pesar de que RubyGems.org no es mantenido por una compañía en concreto, muchas nos han ayudado. El diseño actual, las ilustraciones y el desarrollo de front-end del sitio fue cortesía de %{dockyard}. %{github} también ha sido importante por ayudarnos a colaborar y compartir código de manera sencilla. El sitio comenzó en %{heroku}, cuyo excelente servicio ayudó a probar que RubyGems es una solución viable en la que toda la comunidad puede confiar. Nuestra infraestructura está alojada actualmente en %{aws}.
```

File: config/locales/es.yml [hunk 16]
```diff
@@ -77,254 +82,254 @@ es:
...
+      technical_html: 'Algunos aspectos técnicos del sitio: Es 100% Ruby. El sitio principal es una aplicación de %{rails}. Las gemas se almacenan en %{s3}, son servidas por %{fastly}, y el tiempo entre que se publica una gema y que está lista para instalarse es mínimo. Para más información, %{source_code}, cuya licencia es %{license} en GitHub.'
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      technical_html: 'Algunos aspectos técnicos del sitio: Es 100% Ruby. El sitio principal es una aplicación %{rails}. Las gemas se almacenan en %{s3}, son servidas por %{fastly}, y el tiempo entre que se publica una gema y que está lista para instalarse es mínimo. Para más información, %{source_code}, cuya licencia es %{license} en GitHub.'
```

File: config/locales/es.yml [hunk 17]
```diff
@@ -77,254 +82,254 @@ es:
...
-      success:
+      success: Has actualizado exitosamente tu nivel de MFA.
   profiles:
     edit:
-      email_awaiting_confirmation:
-      enter_password:
-      hide_email: Esconder email en perfil público
-      optional_twitter_username:
+      email_awaiting_confirmation: Por favor confirma tu nueva dirección de correo %{unconfirmed_email}
+      enter_password: Por favor entra tu contraseña
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      enter_password: Por favor introduce tu contraseña
```

File: config/locales/es.yml [hunk 18]
```diff
@@ -77,254 +82,254 @@ es:
...
+      confirm: Confirmar
+      instructions: Lamentamos ver que te vas. Ingresa tu contraseña abajo y confírmalo.
+      list_only_owner_html: Estas gemas serán borradas cuando elimines tu cuenta. Si quieres agregar propietarios antes de eliminar tu cuenta, puedes usar el comando %{command_link}.
+      list_multi_owner: Perderás acceso a estas gemas pero otros propietarios podrán seguir accediendo a ellas.
     rubygem:
-      owners_header:
+      owners_header: Propietarios
     destroy:
-      request_queued:
+      request_queued: Tu solicitud de eliminación de cuenta se ha puesto en cola. Te enviaremos un email de confirmación cuando haya sido procesado.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      request_queued: Tu solicitud de eliminación de cuenta se ha puesto en cola. Te enviaremos un email de confirmación cuando haya sido procesada.
```

File: config/locales/es.yml [hunk 19]
```diff
@@ -77,254 +82,254 @@ es:
...
         subscribe: Suscribirse
-        unsubscribe: Unsubscribe
+        unsubscribe: Desuscribirse
         wiki: Wiki
     blacklisted:
-      blacklisted_namespace:
+      blacklisted_namespace: Este namespace está reservado por rubygems.org.
     dependencies:
-      header: dependencias de %{title}
+      header: "%{title} Dependencias"
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
      header: "dependencias de %{title}"
```

File: config/locales/es.yml [hunk 20]
```diff
@@ -77,254 +82,254 @@ es:
...
       versions_header: Versiones
-      yanked_notice: Esta gema ya no está disponible para descarga directa o para otras gemas que dependan de ella.
+      yanked_notice: Esta versión fue borrada, y no está disponible para su descarga directa ni por otras gemas que puedan haber dependido de la misma.
     show_yanked:
-      not_hosted_notice: Esta gema no se encuentra alojada en Gemcutter.
+      not_hosted_notice: Esta gema no está alojada actualmente en RubyGems.org.
       reserved_namespace_html:
-        one:
-        other:
+        one: Esta gema existió previamente, pero fue eliminada por sus propietarios. El equipo de RubyGems.org ha reservado este nombre por un día más. Luego cualquiera podrá solicitar este nombre de gema usando gem push. <br/> Si eres el anterior propietario, puedes cambiar la propiedad de esta gema usando el comtando gem owner. También puedes crear nuevas versiones de esta gema usando gem push.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
        one: Esta gema existió previamente, pero fue eliminada por sus propietarios. El equipo de RubyGems.org ha reservado este nombre por un día más. Luego cualquiera podrá solicitar este nombre de gema usando `gem push`. <br/> Si eres el anterior propietario, puedes cambiar la propiedad de esta gema usando el comando `gem owner`. También puedes crear nuevas versiones de esta gema usando `gem push`.
```

File: config/locales/es.yml [hunk 21]
```diff
@@ -77,254 +82,254 @@ es:
...
+        other: Esta gema existió previamente, pero fue eliminada por sus propietarios. El equipo de RubyGems.org ha reservado este nombre por %{count} días más. Luego cualquiera podrá solicitar este nombre de gema usando gem push. <br/> Si eres el anterior propietario, puedes cambiar la propiedad de esta gema usando el comtando gem owner. También puedes crear nuevas versiones de esta gema usando gem push.
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]:
```suggestion
        other: Esta gema existió previamente, pero fue eliminada por sus propietarios. El equipo de RubyGems.org ha reservado este nombre por %{count} días más. Luego cualquiera podrá solicitar este nombre de gema usando `gem push`. <br/> Si eres el anterior propietario, puedes cambiar la propiedad de esta gema usando el comando `gem owner`. También puedes crear nuevas versiones de esta gema usando `gem push`.
```

File: config/locales/es.yml [hunk 22]
```diff
@@ -77,254 +82,254 @@ es:
...
   will_paginate:
-    next_label:
+    next_label: Siguiente
     page_gap: "…"
-    previous_label:
+    previous_label: Previa
     page_entries_info:
-      multi_page:
-      multi_page_html:
+      multi_page: Displaying %{model} %{from} - %{to} of %{count} in total
```
Comment (deivid-rodriguez) [inline config/locales/es.yml]: A partir de aquí en adelante está todo en inglés.

**Comment by Lastimoso** [comment]
I did a mess and needed to force a push. I'm reopaning PR with the updated file

**Comment by Lastimoso** [comment]
I apologise @deivid-rodriguez. Your suggestions are included in the file but I lost the co-authored commit. Please propose changes so you are included again. 

@sonalkr132 sorry for the delay. I think this may be a winner.

**Comment by deivid-rodriguez** [comment]
As long as my suggestions are included with the commit, I don't care about the attribution.

**Comment by Lastimoso** [comment]
Hi @sonalkr132, all suggestions from Deivid have been added. Everything is translated and reviewed from my side. Please let me know if there is anything I can do.

**Comment by sonalkr132** [comment]
Sorry about delay @Lastimoso It kinda slipped through.
Also, thank you so much for working on this :heart_eyes_cat:

**Comment by Lastimoso** [comment]
Thanks! My pleasure. Just trying to give back a little.

**Pull request merged**